cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -218,95 +224,85 @@ mkTracers traceOptions tracer = Tracers
 
     mempoolTracer = Tracer $ \ev -> do
       traceWith (mempoolTraceTransformer tracer) ev
-      traceWith (enableConsensusTracer Consensus.mempoolTracer
-                $ withName "Mempool" tracer) ev
-
-    enableConsensusTracer
-      :: Show a
-      => (ConsensusTraceOptions -> Const Bool b)
-      -> Tracer IO String -> Tracer IO a
-    enableConsensusTracer f = if getConst $ f $ traceConsensus traceOptions
-      then showTracing
-      else const nullTracer
-    hasConsensusTraceFlag
-      :: (ConsensusTraceOptions -> Const Bool b)
-      -> Bool
-    hasConsensusTraceFlag f = getConst $ f $ traceConsensus traceOptions
-
-    mkConsensusTracers :: Consensus.Tracers' peer blk (Tracer IO)
-    mkConsensusTracers = Consensus.Tracers
+      traceWith (showTracing $ withName "Mempool" tracer) ev
+
+    mkConsensusTracers :: TraceOptions -> Consensus.Tracers' peer blk (Tracer IO)
+    mkConsensusTracers traceOpts = Consensus.Tracers
       { Consensus.chainSyncClientTracer
-        = annotateSeverity $ filterSeverity (pure . const (tracingSeverity $ hasConsensusTraceFlag Consensus.chainSyncClientTracer))
-          $ toLogObject' (tracingFormatting $ hasConsensusTraceFlag Consensus.chainSyncClientTracer) tracingVerbosity
+        = tracerOnOff (traceChainSyncClient traceOpts)
+          $ annotateSeverity $ filterSeverity (pure . const (tracingSeverity True))
+          $ toLogObject' (tracingFormatting True) tracingVerbosity
```

Comment:
also "tracingFormatting" does not depend on the flag anymore. do we have another cli arg to distinguish between textual and structured output? if not I think we should go for "StructuredLogging" in all cases.

Reply:
@CodiePP at the moment I'm trying to establish a process for adding/removing cli flags, I've talked to @i-o-m about this so for now I will default it to `StructuredLogging`